src/app/code/community/FireGento/GridControl/Model/Processor.php

```diff
@@ -110,9 +110,15 @@ protected function _addAction($params)
                 if (strpos((string) $attribute, '::') !== false) {
                     list($_module, $_method) = explode('::', (string) $attribute);
                     $_module = Mage::getSingleton($_module);
+                    if ($_args = strstr($_method, '(')) {
+                        $_method = str_replace($_args, '', $_method);
+                        $_args = array_map('trim', explode(',', str_replace(array('(', ')', '\'', '"'), '', $_args)));
```

Comment:
Hmm.  Could we use a `<args><arg1>arg1value</arg1></args>` syntax like Magento xml typically does?

I'm not sure I'm picturing the xml structure this supports correctly, though, maybe that would be inconvenient.